test/Microsoft.AspNet.Mvc.Core.Test/Formatters/XmlDataContractSerializerInputFormatterTest.cs

```diff
@@ -338,6 +345,126 @@ public void XmlDataContractSerializerFormatterThrowsWhenMaxDepthIsBelowOne()
             Assert.Equal(expectedString, levelOneModel.sampleString);
         }
 
+        [Fact]
+        public async Task XmlDataContractSerializerFormatterThrowsWhenNotConfiguredWithRootName()
```

Comment:
nit: since these tests are part of `XmlDataContractSerializerInputFormatterTest.cs` file, you could  remove these prefixes (yeah, i think you probably followed other tests which are doing this) as the test name is is not easily readable.